### ISSUE TITLE
Batch download

### ISSUE THREAD

**Issue by leptoid** [seiya-dev/funimation-downloader-nx#3]
Can I make this do all seasons/episodes in 1 command or do I need to make a batch file to perform that task/action sequentially?  Either way works and is easy enough.  I'm way ahead of where I was last week when it came to this process.

**Comment by leptoid** [comment]
I am testing this method for batching:
Step 1:  Figure out show # and episode numbers
Step 2:  Create txt file with the following info:
```
start /wait cmd.exe /c D:\funimation-downloader-nx-master\funi.bat -s ##### --sel #
start /wait cmd.exe /c D:\funimation-downloader-nx-master\funi.bat -s ##### --sel #
start /wait cmd.exe /c D:\funimation-downloader-nx-master\funi.bat -s ##### --sel #
start /wait cmd.exe /c D:\funimation-downloader-nx-master\funi.bat -s ##### --sel #
start /wait cmd.exe /c D:\funimation-downloader-nx-master\funi.bat -s ##### --sel #
start /wait cmd.exe /c D:\funimation-downloader-nx-master\funi.bat -s ##### --sel #
start /wait cmd.exe /c D:\funimation-downloader-nx-master\funi.bat -s ##### --sel #
start /wait cmd.exe /c D:\funimation-downloader-nx-master\funi.bat -s ##### --sel #
start /wait cmd.exe /c D:\funimation-downloader-nx-master\funi.bat -s ##### --sel #
start /wait cmd.exe /c D:\funimation-downloader-nx-master\funi.bat -s ##### --sel #
```

```
##### = Show #
# = Episode number
```
Step 3:  Rename the file extension to .bat
Step 4:  Run the file.

Turns out this works however, main script will require modification to take out the pause after the final step otherwise it gets stuck waiting.

**Comment by seiya-git** [comment]
Bat file must be placed in main directory with following content:
```"node" "%~dp0.\scripts\funidl.js" -s ##### --sel #
"node" "%~dp0.\scripts\funidl.js" -s ##### --sel #
"node" "%~dp0.\scripts\funidl.js" -s ##### --sel #
```
```
##### = Show #
# = Episode ID
```
I'm not working on this right now, but PR are welcome.

**Comment by leptoid** [comment]
No problem.  I don't code much, but I am attempting to contribute / solve my own questions. :)
The board is like scratch paper and group thinking space.

**Comment by seiya-git** [comment]
implemented in 3.0 branch 4521607
